generators/server/templates/pom.xml.ejs

```diff
@@ -923,6 +923,28 @@
             </plugins>
         </pluginManagement>
     </build>
+    <%_ if (addSpringMilestoneRepository) { _%>
+    <repositories>
+        <repository>
+            <id>spring-milestones</id>
+            <name>Spring Milestones</name>
+            <url>https://repo.spring.io/milestone</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>spring-milestones</id>
+            <name>Spring Milestones</name>
+            <url>https://repo.spring.io/milestone</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </pluginRepository>
+    </pluginRepositories>
+    <%_ } _%>
```

Comment:
```suggestion
```

Reply:
How does it get added without it? Just curious for knowledge.